evm/v0.5/src/helpers.ts

```diff
@@ -516,11 +516,39 @@ export function encodeServiceAgreement(sa: ServiceAgreement) {
   )
 }
 
-export function encodeOracleSignatures(os: OracleSignatures) {
+export function encodeOracleSignatures(os: OracleSignature) {
   const osValues = [os.vs, os.rs, os.ss]
   return ethers.utils.defaultAbiCoder.encode(ORACLE_SIGNATURES_TYPES, osValues)
 }
 
+export async function computeOracleSignature(
```

Comment:
Should these exported functions have comments to describe what they do?

Reply:
Tracked here: https://www.pivotaltracker.com/story/show/170938134